Source/SelfService/Web/microservice/purchaseOrder/configuration.tsx

```diff
@@ -301,6 +316,11 @@ export const Configuration: React.FunctionComponent<Props> = (props) => {
     };
 
     const handleBack = () => {
+        if (!isCreate && activeStep === 2) {
+            setNotification('You cannot make changes to the name of the integration type', 'error');
```

Comment:
Looks like a typo (of->or), and maybe change to "ERP system" type since that's what it says in the selector.
```suggestion
            setNotification('You cannot make changes to the name or the ERP system type', 'error');
```

Reply:
Done :)